src/api/api.ts

```diff
@@ -221,7 +223,10 @@ export class GhostApi {
   ): Promise<R> {
     const makeAttempt = (retries: number): Promise<R> => {
       debugLog(`Calling ${init.method} ${url}`);
-      return fetch(url, { ...init, ...{ timeout: 1000 } })
+      return fetch(url, {
+        ...init,
+        ...{ timeout: getenv.int('GHOST_API_FETCH_TIMEOUT_MS', 10000) },
```

Comment:
I think it might be nice to have a file with all the constants so we can easily track them down, avoid duplicates and also document them if needed.

Reply:
no need to wrap the timeout into a new object. Just do

```
{
  ...init,
  timeout: getenv.int('GHOST_API_FETCH_TIMEOUT_MS', 10000) }
}
```